app/views/user/show/otherTrophies.scala

```diff
@@ -17,7 +17,7 @@ object otherTrophies {
           trophy.kind.icon.map { iconChar =>
             a(
               awardCls(trophy),
-              href := trophy.kind.url,
+              href := trophy.kind.url.orElse(trophy.url),
```

Comment:
This is my first issue, in a new code base so please forgive my inexperience with scala.

Conceptually this seems like it should be the other way around, if both the trophy and trophy kind have URLs we'd probably want the trophy url, but that also seemed more likely to break an existing link so I balked and went with this.